src/components/AddPaymentForm.js

```diff
@@ -71,11 +71,12 @@ const AddPaymentForm = (props) => {
             }
             const newPayment = await createPayment({ variables })
             if (loadingNewPayment) return <LoadingProgress/>
-            if (newPayment.errors) {
+            if (errorNewPayment) {
```

Comment:
Let's check if we can move this down to the catch clause so our error handling is cleaner/consistent